fastcrypto-zkp/Cargo.toml

```diff
@@ -39,11 +39,14 @@ schemars ="0.8.10"
 serde = { version = "1.0.152", features = ["derive"] }
 serde_json = "1.0.93"
 once_cell = "1.16"
-poseidon-ark = { git = "https://github.com/arnaucube/poseidon-ark.git", rev = "ff7f5e05d55667b4ffba129b837da780c4c5c849" }
 bcs = "0.1.4"
 im = "15"
 reqwest = { version = "0.11.20", default_features = false, features = ["blocking", "json", "rustls-tls"] }
 rustls-webpki = "0.101.4"
+neptune = { git = "https://github.com/lurk-lab/neptune", rev = "d8b4eeadd8acc9d9e8d9d510605c954f1410aa60" }
```

Comment:
it's now merged in main branch?

Reply:
Yes. I just pinned it to a commit until it's released.